### ISSUE TITLE
The delegator example is successful only on the first compilation.

### ISSUE THREAD

**Issue by ii-ii-ii** [paritytech/ink#693]
Under examples/delegator, execute the command
`. /build-all.sh`
If the contract has not been compiled before, it will compile normally.
If it has been built before, you will get an error.
<img width="1431" alt="image" src="https://user-images.githubusercontent.com/69485494/108653714-10744c00-752c-11eb-99ef-d3fa7e254abc.png">

After deleting the target folder, it will compile normally.

**Comment by ii-ii-ii** [comment]
Any ideas?

**Comment by Robbepop** [comment]
Are you using `cargo-contract` version `9.0`? We released it yesterday. Please see if you can compile it with the new version. Install the new version using `cargo install --force cargo-contract`.

**Comment by atenjin** [comment]
yes, we are using 0.9.0 version now. It can replay, just compile delegator twice.

**Comment by ascjones** [comment]
@cmichi could be related to https://github.com/paritytech/cargo-contract/pull/174?
